src/main/java/org/broadinstitute/dsm/model/ParticipantWrapper.java

```diff
@@ -143,6 +148,15 @@ else if (DBConstants.DDP_KIT_REQUEST_ALIAS.equals(source)) {
                         kitRequests = KitRequestShipping.getKitRequests(instance, filters.get(source));
                         baseList = getCommonEntries(baseList, new ArrayList<>(kitRequests.keySet()));
                     }
+                    else if (DBConstants.DDP_PARTICIPANT_DATA_ALIAS.equals(source)) {
+                        participantData = ParticipantData.getParticipantData(instance.getName(), filters.get(source));
+                        baseList = getCommonEntries(baseList, new ArrayList<>(participantData.keySet()));
+
+                        //if study is AT
+                        if ("atcp".equals(instance.getName())) {
```

Comment:
Is it not better to put this if statement on line 230 to avoid code duplication since this if statement is used on line 217 as well

Reply:
It will only go into L213 if participantData was not filtered for, so if study staff had a default filter which already filtered for participantData we would not check and add default values

Reply:
Oh sorry, did not pay attention on `addDefaultValues` method's last argument that it takes query addition.